src/app/components/elements/Userpic.jsx

```diff
@@ -1,51 +1,43 @@
 import React, {Component, PropTypes} from 'react';
 import {connect} from 'react-redux';
 import shouldComponentUpdate from 'app/utils/shouldComponentUpdate';
-import proxifyImageUrl from 'app/utils/ProxifyUrl';
+import { imageProxy } from 'app/utils/ProxifyUrl';
 
 class Userpic extends Component {
-    static propTypes = {
-        account: PropTypes.string
-    }
 
     shouldComponentUpdate = shouldComponentUpdate(this, 'Userpic')
 
     render() {
-        const {json_metadata, width, height} = this.props
+        const {account, json_metadata} = this.props
         const hideIfDefault = this.props.hideIfDefault || false
 
-        let url = null;
 
         // try to extract image url from users metaData
         try {
             const md = JSON.parse(json_metadata);
-            if(/^(https?:)\/\//.test(md.profile.profile_image)) {
-                const size = (width && width > 48)? '320x320' : '120x120';
-                url = proxifyImageUrl(md.profile.profile_image, size);
-            }
-        } catch (e) {}
-        if(!url) {
-            if(hideIfDefault) {
+            if(!/^(https?:)\/\//.test(md.profile.profile_image) && hideIfDefault) {
                 return null;
             }
-            url = require('assets/images/user.png');
+        } catch (e) {
+            /* eslint-disable-line no-empty */
         }
 
-        const style = {backgroundImage: 'url(' + url + ')',
-                       width: (width || 48) + 'px',
-                       height: (height || 48) + 'px'}
+        const style = {backgroundImage: 'url(' + imageProxy() + `/u/${account}/avatar)` };
 
         return (<div className="Userpic" style={style} />)
     }
 }
 
+Userpic.propTypes = {
+    account: PropTypes.string.isRequired
+}
+
 export default connect(
     (state, ownProps) => {
-        const {account, width, height, hideIfDefault} = ownProps
+        const {account, hideIfDefault} = ownProps
         return {
+            account,
             json_metadata: state.global.getIn(['accounts', account, 'json_metadata']),
```

Comment:
This should be removed as well

Reply:
Based on my above comment, you're incorrect.
> The code indicated there are situations where the user does not have a profile image set, and we do not want to show the default image....

Reply:
We could axe `hideIfDefault` if it's ok that default profiles look a bit uglier. Hell, maybe that would encourage more users to customize theirs. Maybe check w/ Pon. But this feature doesn't cause any extra lookups, so the main benefit is simply less LOC.

Reply:
*Scope Creep* The intent of this issue is to get us using the avatar url, not to change the way the ui presents to the user. Let's leave that to a different issue.